saagieapi/projects/gql_template.py

```diff
@@ -32,6 +32,22 @@
   }}
 """
 
+
+gql_update_global_env_var = """
+   mutation {{
+     saveEnvironmentVariable(
+        environmentVariable: {{
+          id:"{0}",
+          name:"{1}",
+          scope:GLOBAL,
+          value:"{2}",
+          description:"{3}",
+          isPassword:{4}
+        }})
+        {{id}}
+    }}
+"""
+
```

Comment:
Maybe it would be better use something like: 


```
gql_update_env_var = """
mutation($id: UUID, $entityId: UUID, $name: String!, $scope: EnvVarScope!, $value: String, $description: String, 
        $isPassword: Boolean!)  {
            saveEnvironmentVariable(
                entityId: $entityId
                environmentVariable: {
                    id: $id
                    name: $name
                    scope: $scope
                    value: $value
                    description: $description
                    isPassword:$isPassword
        
    }){
        id
    }}
"""
```